examples/touch.c

```diff
@@ -110,13 +113,13 @@ int main(int argc, char *argv[]) {
 		wlr_log(L_ERROR, "Could not start compositor, OOM");
 		exit(EXIT_FAILURE);
 	}
-	state.cat_texture = wlr_render_texture_create(state.renderer);
+	state.cat_texture = wlr_texture_from_pixels(state.renderer,
+		WL_SHM_FORMAT_ARGB8888, cat_tex.width, cat_tex.width, cat_tex.height,
```

Comment:
Stride is used inconsistently, is it necessary here? I don't even know how stride works in this library, what are pixels and how do they work